src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -4507,8 +4542,11 @@ DECLARE_API(DumpAsync)
             {
                 // Otherwise, we only care about AsyncStateMachineBox`1 as well as the DebugFinalizableAsyncStateMachineBox`1
                 // that's used when certain ETW events are set.
-                if (_wcsncmp(itr->GetTypeName(), W("System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1+AsyncStateMachineBox`1"), 79) != 0 &&
-                    _wcsncmp(itr->GetTypeName(), W("System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1+DebugFinalizableAsyncStateMachineBox`1"), 95) != 0)
+                DacpMethodTableData mtdata;
+                auto result = mtdata.Request(g_sos, TO_TADDR(itr->GetMT()));
+                if (mtdata.Request(g_sos, TO_TADDR(itr->GetMT())) != S_OK ||
```

Comment:
Did you mean to call mtdata.Request twice here? It looks like you meant to just check "result" from the first time.

Reply:
Nope, thanks. Will fix.